### PR TITLE
Removes hygiene

### PR DESCRIPTION
**Pull request by k-k-karma**
cancer fuck you quistinnus

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
Get a shower stinkey!

**Comment by t-f-w** [comment]
sure +1

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -267,7 +267,7 @@
 
 #define	HUNGER_FACTOR		0.1	//factor at which mob nutrition decreases
 #define	ETHEREAL_CHARGE_FACTOR	0.12 //factor at which ethereal's charge decreases
-#define	HYGIENE_FACTOR  0.1	//factor at which mob hygiene decreases
+#define	HYGIENE_FACTOR  0	//factor at which mob hygiene decreases
```
Comment (GuyonBroadway) [inline code/__DEFINES/mobs.dm:270]:
```suggestion
#define	HYGIENE_FACTOR  0	//factor at which mob hygiene decreases
#define	HYGIENE_FACTOR_TFW 10	//TFW a stinky
```

**Comment by Manly-Dwarf** [comment]
+1 adds nothing to the game. Might be a good trait, though.

**Comment by Lxtoofonea** [comment]
make it a trait, +1

**Pull request merged**